feature-group-definitions/autofill.dist.yml

```diff
@@ -0,0 +1,19 @@
+# Generated from: autofill.yml
+# Do not edit this file by hand. Edit the source file instead!
+
+name: ":autofill"
+description: "The `:autofill` pseudo-class matches `<input>` elements that have been filled in automatically by the browser."
+spec: https://html.spec.whatwg.org/multipage/semantics-other.html#selector-autofill
+status:
+  baseline: low
```

Comment:
Yes: https://developer.mozilla.org/en-US/docs/Web/CSS/:autofill#browser_compatibility

This was supported with a prefix earlier, but that doesn't count as Baseline.